lib/cocoapods/target/build_settings.rb

```diff
@@ -1246,6 +1292,17 @@ def user_target_xcconfig_values_by_consumer_by_key
           merged_xcconfigs(user_target_xcconfig_values_by_consumer_by_key, :user_target_xcconfig)
         end
 
+        # @see BuildSettings#should_apply_xctunwrap_fix?
+        def should_apply_xctunwrap_fix?
+          target.library? && target.user_targets.any? do |ut|
```

Comment:
can we check the platform on the target definitions instead?

Reply:
oh sure